### ISSUE TITLE
7.11.1: Newer version of PHPMailer is not compatible with Email:email2Send method

### ISSUE THREAD

**Issue by dawid-zaroda** [salesagility/SuiteCRM#6932]
SuiteCRM 7.11 comes with newer version of PHPMailer. Method **addAttachment** in PHPMailer object got new condition: **isPermittedPath** which blocks all pathes with ://

Email:emal2Send for $request['documents'] and $request['templateAttachments'] uses pathes:
`$fileLocation = "upload://{$GUID}";`

which will be blocked by isPermittedPath.

`SugarPHPMailer encountered an error: Could not access file: upload://c2a5ebd1-5a6d-0f2f-eafe-5c6286a20e74`
#### Your Environment
* SuiteCRM Version used: 7.11.1
* Environment name and version (e.g. MySQL, PHP 7): PHP 7.1
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04.4 LTS

**Comment by Mausino** [comment]
Is your issue related to #6423 or duplicate of it ?

**Comment by dawid-zaroda** [comment]
I think no, because there: https://github.com/salesagility/SuiteCRM/blob/v7.8.22/include/phpmailer/class.phpmailer.php
is still older version of PHPMailer, which works fine with issue reported by me. This file is not present in 7.11.1

**Comment by Dillon-Brown** [comment]
Thought I had fixed this with #6791 but I might have missed a few places.

**Comment by Mausino** [comment]
I did those changes #7154, did R&R in admin and still if the email template has attachment and is used in email... the attachment isn't added to sent email. Also in compose view isn't visible that attachment was added from Email Template
